highfive/newpr.py

```diff
@@ -386,6 +398,9 @@ def new_pr(payload, user, token):
     if warnings:
         post_comment(warning_summary % '\n'.join(map(lambda x: '* ' + x, warnings)), owner, repo, issue, user, token)
 
+    if "new_pr_labels" in config and config["new_pr_labels"]:
```

Comment:
I'm kind of late now, but you could have done:
```python
if config.get('new_pr_labels'):
```

Empty lists are falsy, so I think it would have provided the logic you wanted.

Reply:
Yeah, my Python is a bit... rusty.